dask_distance/__init__.py

```diff
@@ -216,27 +216,23 @@ def squareform(X, force="no"):
 
         X_tri = []
         j1 = 0
-        for j2 in _pycompat.irange(d - 1, -1, -1):
+        for j2 in _pycompat.irange(d - 1, 0, -1):
             X_tri.append(X[j1:j1 + j2])
             j1 += j2
```

Comment:
As a side note, we are now able to skip inclusion of this zero length 1-D array. 😄